src/report.ts

```diff
@@ -93,10 +100,19 @@ export class ReportModule {
         return (error as ApiResponseError).response !== undefined;
     }
 
-    private convertFilter(filter: AFM.CompatibilityFilter): AFM.CompatibilityFilter | null {
-        if ((filter as AFM.IExpressionFilter).value === undefined) {
-            return convertVisualizationObjectFilter(filter as VisualizationObjectFilter);
+    private sanitizeFilters(filter: AFM.CompatibilityFilter): ExecuteAFM.CompatibilityFilter | null {
```

Comment:
this should be named sanitizeFilter (without s), because it transforms only one filter

Reply:
updated. thanks :)